src/api/api.js

```diff
@@ -0,0 +1 @@
+export const userApi  = 'https://jsonplaceholder.typicode.com/users';
```

Comment:
u shoud create variable for API like this 'https://jsonplaceholder.typicode.com' and use it with adding some additional ways

But api  shoud be created  in `.env` file
.env file shoud be in gitignore.
But for this project u can leave it here